.github/workflows/linter.yml

```diff
@@ -0,0 +1,24 @@
+name: Node Linter
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js v20.16.0
+        uses: actions/setup-node@v4
+
+      - name: Run ESLint
```

Comment:
This step should be a npm install

Reply:
Replaced with npm install and introduced linter ruleset from google as well as recommended eslint rules